usaspending_api/spending_explorer/v2/views/spending_explorer.py

```diff
@@ -17,6 +17,6 @@ def post(self, request):
         filters = json_request.get('filters', None)
 
         # Returned filtered queryset results
-        results = type_filter(_type, filters, limit=SPENDING_EXPLORER_LIMIT)
```

Comment:
Just for clarification, this is to remove the initial work to put a cap on the spending explore until the stakeholders make their decision?